test/mobile/test_lite_script_module.py

```diff
@@ -1,12 +1,26 @@
 import torch
 import torch.utils.bundled_inputs
 import io
-from typing import Dict, List, NamedTuple
+from typing import Dict, List, NamedTuple, Type
 from collections import namedtuple
 import inspect
 
 from torch.jit.mobile import _load_for_lite_interpreter, _export_operator_list
 from torch.testing._internal.common_utils import TestCase, run_tests
+from torch.testing._internal.common_quantized import (
+    override_quantized_engine,
+)
+from torch.testing._internal.common_quantization import (
+    AnnotatedSingleLayerLinearModel,
+    TwoLayerLinearModel,
+    AnnotatedNestedModel
+)
+from torch.quantization import (
+    quantize,
+    # prepare,
```

Comment:
@supriyar and @jerryzh168 , please confirm that if quantize is tested we don't need to use prepare + convert, because they are already used inside quantize.

Reply:
Yes, this should be fine. Do we want to test FX APIs as well?

Reply:
@iseeyuan we need to check tests from https://github.com/pytorch/pytorch/blob/master/test/quantization/fx/test_quantize_fx.py